src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl regService;
+
+    @BeforeAll
+    static void initialize() {
+        regService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void loginIsAlreadyInStorage_NotOk() {
+        User user1 = new User("Alice", "123456", 19);
+        User user2 = new User("Alice", "654321", 22);
+        assertNotNull(regService.register(user1));
+        assertNull(regService.register(user2));
```

Comment:
If user already registered you code must throw exception, and you must check this. Not null value. 
Returning null value, it`s you special implementation. 
Condition was be about exception's 
![image](https://user-images.githubusercontent.com/95774539/196129684-4159624d-18bb-4e69-b86b-2858936e9e1b.png)